libs/ballot-encoder/.eslintrc.js

```diff
@@ -10,7 +10,6 @@ module.exports = {
   },
   parser: '@typescript-eslint/parser', // Specifies the ESLint parser
   extends: [
-    'airbnb-base',
```

Comment:
Is this what adds the rules for the console logs and stuff that you removed the ignores for?

Reply:
Yes. I have been removing the Airbnb config from our projects as I make major changes because I disagree with a lot of their choices.